tasks/simple-batch-job/README.adoc

```diff
@@ -0,0 +1,235 @@
+:sectnums:
+= Batch Job on Cloud Foundry
+
+In this demonstration, you will learn how to orchestrate short-lived data processing application (_eg: Spring Batch Jobs_) using http://cloud.spring.io/spring-cloud-task/[Spring Cloud Task] and http://cloud.spring.io/spring-cloud-dataflow/[Spring Cloud Data Flow] on Cloud Foundry.
+
+== Using Cloud Foundry Server
+
+=== Prerequisites
+
+In order to get started, make sure that you have the following components:
+
+* Local https://pivotal.io/pcf-dev[PCFDev] instance 
+* Local install of https://github.com/cloudfoundry/v3-cli-plugin[V3-CF-CLI] plugin
+* Local build of https://github.com/spring-cloud/spring-cloud-dataflow[Spring Cloud Data Flow]
+* Local build of Spring Cloud Data Flow's https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry[Cloud Foundry Server]
+* Running instance of mysql in PCFDev
+
+NOTE: PCF 1.7.12 or greater is required to run Tasks on Spring Cloud Data Flow. As of this writing, PCFDev and PWS supports builds upon this version.
+
+=== Running the Sample in Cloud Foundry
+
```

Comment:
Once we get into this level, the sample might look too big. For this reason, we don't include these instructions in other (CF-server) samples, too. 

I think it'd be better to leave that as user responsibility. I will add a note about it just so that it is clear.